CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- When running on Windows or Linux, the SDK now links errors and events originating on different layers (managed, native errors) via `trace ID` ([#2089](https://github.com/getsentry/sentry-unity/pull/2089)) 
```

Comment:
question: Could we merge both entries into a single entry linking both PRs, or do we prefer to have one entry per PR?